src/protocol/templates/ping1d.py.in

```diff
@@ -3,26 +3,31 @@
 # Ping1D.py
 # A device API for the Blue Robotics Ping1D echosounder
 
-#~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!
+# ~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!
 # THIS IS AN AUTOGENERATED FILE
 # DO NOT EDIT
```

Comment:
Are you supposed to edit this file?

Reply:
yes, this is the template generating the autogenerated file.

Reply:
@patrickelectric 

Not at all, I just like reading code to try and learn from it.

Reply:
Sorry for asking, but there is some new bots in github and I was curious about your virtual existence, hehe.
Thank you for your inputs, and be welcome to be around.

Reply:
@patrickelectric 

No problem at all! I was thinking it was the short amount of time from when the PR was submitted to when I commented which caused you to think that but let me explain. I go to this [url](https://github.com/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+language%3Apython) and refresh the page to look at all the new pull requests for Python. It was purely random as to me coming across your repo today!

Reply:
@patrickelectric 

Nice to meet you too! See you around!

Reply:
@42B found us out, this text should not appear at the top of this file (the edit-able template), but should be inserted by the script that generates the result.

Reply:
https://github.com/bluerobotics/ping-protocol/issues/107